shell/edit/workload/index.vue

```diff
@@ -922,7 +922,7 @@ export default {
             :label="t('workload.replicas')"
           />
         </div>
-        <div v-if="isStatefulSet" class="col span-4">
+        <div v-if="isReplicable && isStatefulSet" class="col span-3">
```

Comment:
By definition, a StatefulSet is always replicable.

Reply:
I made a tweak, pasting the commit description

- This previously matched the original check for both isReplicable && isStatefulSet
- These work off of the `type` field which does not overlap between types
- It looks like the field `spec.serviceName` is only application to type StatefulSets
  - see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#statefulset-v1-apps

So now the field is only shown for the correct workload type